### DISCUSSION TITLE
Version 1.18.0 Download link ?

### DISCUSSION THREAD

**Discussion by andyweiss7** [WarmUpTill/SceneSwitcher#546] [General]
Hello, i can't download the last version of "Advanced Scene Switcher 1.18.0"

When i go here -> https://github.com/WarmUpTill/SceneSwitcher/releases/tag/1.18.0

and scroll down i only see the versions 1.0.0

Best regards.

**Comment by nacalai** [comment]
It just means version 1.0 of plugin to obs v28. All plugins have to be updated for v28, which is why many devs now have separate releases for those using v28+ and those using v27 and below and also decide to call the v28 plugins for "new" and use 1.0. So, just download according to your OBS version

**Comment by WarmUpTill** [comment]
That "1.0.0" part will be removed in future releases.
Adding this was not intended (was a left over from migration to the new obs plugin template version).

**Comment by devinhedge** [comment]
@WarmUpTill You've done such great work with this plug-in. We really appreciate how hard it is to maintain a plug-in for a larger piece of software. Thank you!

I know it's a hassle, but can you use canonical versioning where v1.x.y is associated with OBS v27.x and v2.x.y is associated with OBS v28.x? And use (alpha) and (beta) in the nomenclature for works in progress? I'd really make a big difference for those of us that have one setup with OBS v27.x and OBS v28.x. Thoughts? Reactions?

**Reply by WarmUpTill** [reply]
Thanks for the suggestion.

I see what you mean, but the incompatibility to older OBS versions was not introduced due to a change on the plugin's side, but rather a jump in the version of the Qt framework used by OBS.

The plugin itself is still compatible and can be compiled for older OBS versions / Qt framework versions.
Even OBS 28 itself can also still be built using Qt 5 and I _believe_ it is still used on some Linux distributions with Qt 5.

So, at least in my opinion, a jump in the major version of the plugin would be incorrect here.

Also note, a similar situation (upgrade of the Qt framework version of OBS) also occurred in the past, but only affected the MacOS version of OBS.
This would in turn then either result in the situation of the releases for the different OS drifting apart in regards to the version number or having major version changes without any breaking changes for some for some of the releases.

But I am the first to admit that my current use of the versioning is not really consistent and could be improved upon.
I usually base this simply on the amount of changes and additions compared to the last version.
(E.g. adding a big new feature => jump in minor version - only fixes and minor changes => jump in patch version)
